includes/functions/export.php

```diff
@@ -101,8 +101,8 @@ function espresso_event_export($ename) {
 			$sql .= " JOIN " . EVENTS_CATEGORY_TABLE . " c ON  c.id = r.cat_id ";
 		}
 
-		$sql .= (isset($_POST['event_status']) && $_POST['event_status'] != '' && $_POST['event_status'] != 'IA') ? " WHERE event_status = '" . $_POST['event_status'] . "' " : " WHERE event_status != 'D' ";
-		$sql .= isset($_REQUEST['category_id']) && $_REQUEST['category_id'] != '' ? " AND c.id = '" . $_REQUEST['category_id'] . "' " : '';
+		$sql .= (isset($_POST['event_status']) && $_POST['event_status'] != '' && $_POST['event_status'] != 'IA') ? " WHERE event_status = '" . sanitize_text_field($_POST['event_status']) . "' " : " WHERE event_status != 'D' ";
+		$sql .= isset($_REQUEST['category_id']) && sanitize_text_field($_REQUEST['category_id']) != '' ? " AND c.id = '" . $_REQUEST['category_id'] . "' " : '';
```

Comment:
category_id is expected to be an int right? if so `(int)` would probably be better here

Reply:
and should probably validate the $_REQUEST variable that's actually used in the query

Reply:
😅 oups! Ya this is totally just sanitizing the value used in the conditional, NOT the value used in the query. Thanks